neo/Ledger/MemoryPool.cs

```diff
@@ -17,8 +17,7 @@ namespace Neo.Ledger
     public class MemoryPool : IReadOnlyCollection<Transaction>
     {
         // Allow a reverified transaction to be rebroadcasted if it has been this many block times since last broadcast.
-        private const int BlocksTillRebroadcastLowPriorityPoolTx = 30;
-        private const int BlocksTillRebroadcastHighPriorityPoolTx = 10;
+        private const int BlocksTillRebroadcast = 10;
```

Comment:
How was decided this value?

Reply:
I do not remember, @jsolman did this improvement.

I picked the high priority now, since there are no more low priority txs.

Reply:
It's good for me, I was just curious

Reply:
I think it was Empirical Experimentation. :D aheyuahuea